unstructured/partition/email.py

```diff
@@ -33,7 +33,6 @@
 
 
 def _parse_received_data(data: str) -> List[Element]:
-
```

Comment:
Did a linting standard change or something?

Reply:
Yeah this was due to the `black` version bump.